src/pages/CodecovAIPage/LearnMoreBlurb/LearnMoreBlurb.tsx

```diff
@@ -0,0 +1,27 @@
+import A from 'ui/A'
+import { Card } from 'ui/Card'
+
+// TODO: Update link to docs once they are available
```

Comment:
Do you already have a ticket for this?

Reply:
https://github.com/codecov/engineering-team/issues/2547